pypose/lietensor/lietensor.py

```diff
@@ -319,13 +334,13 @@ def matrix(self, input):
     def rotation(self, input):
         return input.Exp().rotation()
 
-    def Jr(self, x):
+    def Jr(self, X):
         """
         Right jacobian of so(3)
         """
-        K = vec2skew(x)
-        theta = torch.linalg.norm(x, dim=-1, keepdim=True).unsqueeze(-1)
-        I = torch.eye(3, device=x.device, dtype=x.dtype).expand(x.lshape+(3, 3))
+        K = vec2skew(X)
+        theta = torch.linalg.norm(X, dim=-1, keepdim=True).unsqueeze(-1)
+        I = torch.eye(3, device=X.device, dtype=X.dtype).expand(X.lshape+(3, 3))
```

Comment:
We use `x` to denote Lie algebra, `X` for Lie group.

Reply:
I refactored the argument name because by standard, the overriding method should have exactly the same function signature as the base method.


Consider this situation: If the base method uses `X`

```python
class LieType:
    def Jr(self, X) -> LieTensor: ...
```

and the overriding method has a different parameter name `X -> x`.

```python
class SE3_Type(LieType):
    def Jr(self, x) -> LieTensor: ...
```

Then when someone uses a keyword argument (say `some_type.Jr(X= ...)`), and the `some_type` is an instance of `SE3_Type`, an unexpected exception will be raised (as there is no argument with name `X` in `SE3_Type.Jr`)

Reply:
This explanation is acceptable. Thank you.